docs/book/src/SUMMARY.md

```diff
@@ -45,12 +45,10 @@
 
 - [Migrations](./migrations.md)
 
-  - [Kubebuilder v1 vs v2](./migration/v1vsv2.md)
+  - [Kubebuilder v2 vs v3](./migration/v2vsv3.md)
```

Comment:
Hi @estroz, 

IMO we should not remove the V1 to the V2 migration guide.  It should still
What do you need is add a new migration V2 to V3 instead.